spec/cassia/access_controller_spec.rb

```diff
@@ -0,0 +1,121 @@
+require 'spec_helper'
+
+RSpec.describe Cassia::AccessController do
+  include FaradayHelpers
+  
+  describe "#get_token" do
+    vcr_options = { cassette_name: 'access_controller/get_token/success', record: :new_episodes }
+      context "when successful", vcr: vcr_options do
+        it "sets the access_token" do
+          access_controller = described_class.new
+          response_body = {"token_type"=>"bearer","access_token"=>"e18a98abddaf5a241dd528a0beaae9b387fd44a02efa01d4b7d71f30e7dc4456","expires_in"=>3600}
+          response = build_response(status: 200, body: response_body)
+          response_handler = Cassia::ResponseHandlers::GetToken.new(access_controller)
```

Comment:
this method should be testing `get_token` on the `AccessController` class but we are not actually calling that function in this test.

The test should look something like
```
access_controller = described_class.new

access_controller.get_token

expect(access_controller.access_token).not_to be_nil
```

And then the same thing for the following tests in this spec.

We should actually call the function we want to test and use cassettes instead of building the responses ourselves